### PR TITLE
Use prop-types package

### PR DESCRIPTION
**Pull request by calvinl**
As of React v16.0.0, using PropTypes in the main React package is deprecated and has been removed. Using this package in conjunction with React 16.0.0 causes build errors.

### CODE REVIEW

**Comment by calvinl** [comment]
@nicolaslopezj Could you please take a look at this? It's the only package in my list that's breaking with React 16. Would appreciate a new version!

Thanks!

**Review by nguyennb9 — APPROVED** [review]

**Comment by Antbrooksuk** [comment]
Please complete this. @nguyennb9

**Comment by nguyennb9** [comment]
@Antbrooksuk I'm just have permission to review, the author needs to merge this pull request

File: package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
     "react": "^15.4.1"
   },
   "dependencies": {
-    "autobind-decorator": "^1.3.4"
+    "autobind-decorator": "^1.3.4",
+    "prop-types": "^15.6.0"
```
Comment (felquis) [inline package.json]: We should also test it with react 16, react and prop-types deps should have the same version.

**Comment by calvinl** [comment]
I think we should fork this and publish a new npm package for this if enough of us need it. Personally, I’ve just created my own version of this locally in my project. 

The author seems to be MIA and I don’t think any of us has merge/publish permissions.

**Comment by felquis** [comment]
@calvinl totatally, someone already did that? Is there a fork avaiable? 

1 - publish a fork of this repo under react-linkedin-auth
2 - keep a channel about it open on Telegram so we can easily find the maintainer
3 - create an issue here mentioning the fork
4 - put 4 people as colaborators who can audit and merge PRs

So lets just publish it under react-linkedin-auth, but lets keep people close enough, so PRs like this one wont die again.

The repo owner is probably very focused on something else, we should try to conc.... wait it is mantained by a private company, that is why nobody saw this issue, they have a lot of repositories.

I sent an e-mail, lets try something like this too: @bermann @d-a-g-e @diegoolavarria @hansfpc @joadr @nicolaslopezj @Uner4s

**Comment by nicolaslopezj** [comment]
Published at 1.0.4

**Comment by hiphigi** [comment]
For some reason it gives me the error that it's unable to find the module after updating to 1.0.4. Is the endpoint correctly placed?

**Comment by nicolaslopezj** [comment]
It's ```react-linkedin-login```, not ```auth```

**Comment by hiphigi** [comment]
The issue occurs if you are running React 16.* as this requires React 15.* - care to update it for React 16?

**Pull request merged**